devito/core/gpu_openmp.py

```diff
@@ -417,4 +505,7 @@ def _specialize_iet(cls, graph, **kwargs):
         data_manager.place_definitions(graph)
         data_manager.place_casts(graph)
 
+        # Initialize OpenMP environment
+        initialize(graph)
```

Comment:
same as before -- this should be at the top of the pipeline

Reply:
Doing that generates incorrect C code. Also, that is the same position `initialize` is in `gpu_openacc.py`.